wordpress-mobile/WordPress-iOS
# Media Library: Improve loading state

This PR improves the appearance of the media library when a user first visits their Media Library. Currently, they'll see the no results view saying "You have no media" even if they do have some. This PR adds a new 'fetching media...' animated no results view to keep the user better informed about what's happening when we're fetching media the first time.

I had to add two new delegate methods to `MediaPicker-iOS` to let me update the state correctly. You can find that here: https://github.com/wordpress-mobile/MediaPicker-iOS/pull/179

This PR currently points the Podfile at that branch. Once that's been merged and released, I'll update the pod file here before merging. I've set the 'do not merge' label as a reminder.

To test:

* Build and run
* Visit the Media Library section for a site with no media, or a site you haven't yet viewed in the app
* Check that when the media section first loads, you see 'Fetching media..." displayed
* Check that when you upload the first media item to an empty library, the no results view is removed from view.

Needs review: @kurzee
